modules/installation-ibm-z-user-infra-machines-iso.adoc

```diff
@@ -77,22 +77,23 @@ rd.znet=qeth,0.0.bdf0,0.0.bdf1,0.0.bdf2,layer2=1,portno=0 zfcp.allow_lun_scan=0
 +
 [NOTE]
 ====
-If additional LUNs are configured with NPIV, FCP requires `zfcp.allow_lun_scan=0`.
+If additional LUNs are configured with NPIV, FCP requires `zfcp.allow_lun_scan=0`. If you need to enable `zfcp.allow_lun_scan=1` because you are using a CSI driver, for example, you must configure your NPIV so that each node cannot access the boot partition of another node.
 ====
 ... Leave all other parameters unchanged.
 +
 [NOTE]
 ====
-Additional post-installation steps are required to fully enable multipathing. See “Completing installation on user-provisioned infrastructure” for more information.
+Additional post-installation steps are required to fully enable multipathing. For more information, see “Enabling multipathing with kernel arguments on {op-system}" in _Post-installation machine configuration tasks_.
```

Comment:
Consider just saying this here:
```suggestion
Additional post-installation steps are required to fully enable multipathing.
```

And adding an xref in the `.Next steps` section of the assembly.

I also suggest changing the NOTE to IMPORTANT.